etc/docker/elyra/start-elyra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export NB_PORT=${NB_PORT:-8888}
+export GATEWAY_HOST=${GATEWAY_HOST:-localhost}
+export KG_URL=${KG_URL:-http://${GATEWAY_HOST}:${NB_PORT}}
```

Comment:
Use of the `KG_` envs is obsolete - assuming you're only supporting the embedded gateway.  `KG_` should be replaced with `JUPYTER_GATEWAY_`.

Reply:
Yes, `JUPYTER_GATEWAY...` is being set on the jupyter hub config.

Reply:
Ok - thanks.  I guess that explains the comment below.  Why are we dealing with GATEWAY_HOST and KG_URL at all then if JUPYTER_GATEWAY_URL is already set?